apps/frontpage/app/versions/route.ts

```diff
@@ -1,44 +1,33 @@
-import { stringify } from 'node:querystring';
 import crypto from 'node:crypto';
-import fetch from 'node-fetch';
-import LRU from 'lru-cache';
+import { headers } from 'next/headers';
+import type { NextRequest } from 'next/server';
 import { BigQuery } from '@google-cloud/bigquery';
-import semver from 'semver';
-
-const CACHE_KEY = 'versions';
-const cache = new LRU({
-  max: 64 * 1024, // 64kb
-  maxAge: 10 * 60 * 60 * 1000, // 10 mins
-  length: (n, key) => n.length,
-});
```

Comment:
According to [Next.js docs](https://nextjs.org/docs/app/building-your-application/routing/route-handlers#caching), caching is built in to route handlers like this one.